R/html_vignette.R

```diff
@@ -18,7 +20,9 @@
 #' documentation} for additional details on using the \code{html_vignette} format.
 #'
 #' @inheritParams html_document
-#' @param ... Additional arguments passed to \code{\link{html_document}}
+#' @param ... Additional arguments passed to \code{\link{html_document}}. Please
+#'   note that \code{theme}, \code{fig_retina} and \code{highlight} are hard
+#'   coded. Setting them will yield an error.
```

Comment:
Typo `them` -> `theme`.

Actually setting any of these hard-coded arguments will yield errors (it is not only `theme`).